mos_tests/neutron/python_tests/test_ovs_restart.py

```diff
@@ -0,0 +1,161 @@
+#    Copyright 2015 Mirantis, Inc.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+
+import time
+
+import pytest
+
+from mos_tests.neutron.python_tests.base import TestBase
+
+
+class OvsBase(TestBase):
+    """ Common fuctions for ovs tests"""
+
+    def setup_rules_for_default_sec_group(self):
+        """Add necessary rules to default security group."""
+        default_sec_group = [
+            group for group in self.os_conn.nova.security_groups.list()
+            if group.name == "default"][0]
+
+        self.os_conn.nova.security_group_rules.create(
+            default_sec_group.id,
+            ip_protocol='icmp',
+            from_port=-1,
+            to_port=-1,
+            cidr='0.0.0.0/0')
+        self.os_conn.nova.security_group_rules.create(
+            default_sec_group.id,
+            ip_protocol='tcp',
+            from_port=1,
+            to_port=65535,
+            cidr='0.0.0.0/0')
+        self.os_conn.nova.security_group_rules.create(
+            default_sec_group.id,
+            ip_protocol='udp',
+            from_port=1,
+            to_port=65535,
+            cidr='0.0.0.0/0')
+
+    def disable_ovs_agents_on_controller(self):
+        """Disable openvswitch-agents on all controllers."""
+        controller = self.env.get_nodes_by_role('controller')[0]
+        assert controller is not None, 'No controllers have been found.'
+
+        with self.env.get_ssh_to_node(controller.data['ip']) as remote:
+            result = remote.execute(
+                '. openrc && pcs resource disable '
+                'p_neutron-plugin-openvswitch-agent --wait')
+            assert result['exit_code'] == 0
+
+    def restart_ovs_agents_on_computes(self):
+        """Restart openvswitch-agents on all computes."""
+        computes = self.env.get_nodes_by_role('compute')
+        assert len(computes) > 0, 'No computes have been found.'
```

Comment:
Let's make this check in mark or test class ot method
`@pytest.mark.check_env_('smth')`